surfer/viz.py

```diff
@@ -2662,44 +2757,85 @@ def _scale_mayavi_lut(lut_table, fmin, fmid, fmax, transparent,
     """
     if not (fmin < fmid) and (fmid < fmax):
         raise ValueError("Invalid colormap, we need fmin<fmid<fmax")
+    if not (alpha >= 0) and (alpha <= 1):
+        raise ValueError("Invalid alpha: it needs to be within [0, 1]")
 
     # Cast inputs to float to prevent integer division
     fmin = float(fmin)
     fmid = float(fmid)
     fmax = float(fmax)
 
-    logger.info("colormap: fmin=%0.2e fmid=%0.2e fmax=%0.2e "
-                "transparent=%d" % (fmin, fmid, fmax, transparent))
+    divergent = center is not None
+
+    if divergent:
+        logger.info("colormap: fmin=%0.2e fmid=%0.2e fmax=%0.2e "
```

Comment:
How about messages formatted more like this (aiming to be compact + readable):
```
colormap divergent: center=0., [2., 3., 4.] (transparent)
colormap sequential: [0, 5, 10] (opaque)
```

Reply:
but is there a convention that you have been using for these messages, or does anything depend on their previous format?

Reply:
I think it's okay to change it to make it clearer / more compact, it is very unlikely someone's code relies on parsing the colormap log message